src/metadata/debug_utils.rs

```diff
@@ -266,3 +317,8 @@ extern "C" fn print_i1_impl(value: bool) {
 extern "C" fn print_pointer_impl(value: *const ()) {
     println!("[DEBUG] {value:018x?}");
 }
+
+extern "C" fn print_pointer_felt252(l0: u32, l1: u32, l2: u32, l3: u32) {
```

Comment:
I think you want `u64` instead of `u32`.
```suggestion
extern "C" fn print_pointer_felt252(l0: u64, l1: u64, l2: u64, l3: u64) {
```

Reply:
Can't create a biguint from u64, any idea how?

Reply:
Yes, `from_le_bytes()`.

Reply:
Only accepts &[u8]

Reply:
Yes, it should accept `&[u8]`. You just need to convert the four limbs into a byte array.
For example:
```rs
BigUint::from_bytes_le(
    &l0.to_le_bytes()
        .into_iter()
        .chain(l1.to_le_bytes())
        .chain(l2.to_le_bytes())
        .chain(l3.to_le_bytes())
        .collect::<Vec<_>>(),
)
```

Reply:
Thats amazing, thank you so much!!!